bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostAsterism.scala

```diff
@@ -0,0 +1,368 @@
+package edu.gemini.spModel.gemini.ghost
+
+import edu.gemini.spModel.core.{Coordinates, Magnitude, MagnitudeBand, MagnitudeSystem, Offset}
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.spModel.target.env.Asterism
+
+import java.time.{Duration, Instant}
+
+import scalaz._
+import Scalaz._
+
+/** Base trait for the three GHOST asterism types: two target, beam switching,
+  * and high resolution.
+  */
+sealed trait GhostAsterism extends Asterism {
+  import GhostAsterism.{XBinning, YBinning}
+
+  def xBinning: XBinning
+  def yBinning: YBinning
+}
+
+/** GHOST asterism model.
+  */
+object GhostAsterism {
+
+  /** The GHOST guide fibers can be enabled or disabled for each science target.
+    * Typically they are enabled for bright targets (< B mag 18 for standard
+    * resolution, < B mag 17 for high resolution) but disabled for faint targets.
+    * OIWFS state can also be explicitly disabled, say, in a crowded field where
+    * OIWFS is less effective.
+    */
```

Comment:
You're right.  I'm not sure this counts as an OIWFS and would probably be better named `GuideState` or `GuideFiberState`.  I'll change it.